src/fabric/src/fabric_view_all_docs.erl

```diff
@@ -273,10 +313,24 @@ handle_message({execution_stats, _} = Msg, {_, From}, St) ->
     rexi:stream_ack(From),
     {Go, St#collector{user_acc = Acc}}.
 
-merge_row(fwd, Row, Rows) ->
+insert_row(_Dir, _Key, R, []) ->
+    [R];
+insert_row(fwd, Key, R, [H | T] = List) ->
+    V1 = maps:get(Key, R),
+    V2 = maps:get(Key, H),
```

Comment:
I think we can just use `id` here and also do a more common KV match `#{id := RowId} = R`and `#{id := HeadId} = H`. With `id` atom those an be matched directly in the head, whichever one looks better to you.